pages/api/auth/[...nextauth].ts

```diff
@@ -11,6 +14,14 @@ export default NextAuth({
             server: process.env.EMAIL_SERVER,
             from: process.env.EMAIL_FROM,
         }),
+         GithubProvider({
+            clientId: process.env.GITHUB_ID,
```

Comment:
Hi @mikah13 Could you adjust a bit here? It should check the value from .env file before adding to the options. What do you think?

Reply:
I just think we can let users handle that as they can simple comment/remove if they want to opt-out of those options. Although, I can put that in the README so that they are aware.

Reply:
You could see it's broken the current build process https://github.com/jellydn/next-app-starter/pull/1362#issuecomment-1767376866 
```sh

./pages/api/auth/[...nextauth].ts:18:13
--
08:03:38.835 | Type error: Type 'string \| undefined' is not assignable to type 'string'.
08:03:38.835 | Type 'undefined' is not assignable to type 'string'.
08:03:38.835 |  
08:03:38.835 | 16 \|         }),
08:03:38.835 | 17 \|          GithubProvider({
08:03:38.835 | > 18 \|             clientId: process.env.GITHUB_ID,
08:03:38.835 | \|             ^
08:03:38.835 | 19 \|             clientSecret: process.env.GITHUB_SECRET,
08:03:38.835 | 20 \|         }),
08:03:38.835 | 21 \|           GoogleProvider({
08:03:38.930 | error Command failed with exit code 1.

```